src/circuit/gadgets/data.rs

```diff
@@ -205,6 +213,9 @@ impl<F: LurkField> GlobalAllocations<F> {
         let false_num = allocate_constant(&mut cs.namespace(|| "false"), F::zero())?;
         let default_num = allocate_constant(&mut cs.namespace(|| "default"), F::zero())?;
 
+        let power2_64_ff = F::from_str_vartime("18446744073709551616").unwrap();
```

Comment:
It might be clearer to compute this, for example using `pow_vartime()`.

Reply:
done

Reply:
Maybe this is a GitHub anomaly, but this looks unchanged as of now.

Reply:
my bad

Reply:
fixed